launcher/ui/MainWindow.cpp

```diff
@@ -1337,6 +1337,20 @@ void MainWindow::on_actionDeleteInstance_triggered()
     if (response != QMessageBox::Yes)
         return;
 
+    auto linkedInstances = APPLICATION->instances()->getLinkedInstancesById(id);
+    if (!linkedInstances.empty()) {
+        response = CustomMessageBox::selectable(
+                this, tr("There are linked instances"),
+                tr("The folowing instance(s) might reference files in this instance:\n\n"
+                "%1\n\n"
+                "Deleting it could break the other instance(s), \n\n"
+                "Do you wish to proceed?").arg(linkedInstances.join("\n")), 
```

Comment:
```suggestion
                tr("The following instance(s) might reference files in this instance:\n\n"
                "%1\n\n"
                "Deleting it could break the other instance(s), \n\n"
                "Do you wish to proceed?", nullptr, linkedInstances.count()).arg(linkedInstances.join("\n")), 
```

Ok, I actually don't know for sure this is how it's done, but my interpretation of https://doc.qt.io/qt-6/i18n-source-translation.html#handle-plural-forms says this is how you do it :cold_sweat: